### PR TITLE
[VEN-2341] Decrease XVS Emissions

### PR DESCRIPTION
**Pull request by web3rover**
## Description

Resolves VEN-2341

### CODE REVIEW

File: vips/vip-245.ts [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+    oldBorrowSpeed: emissionPerBlock("1.56").div(2).toString(),
+  },
+  {
+    name: "vTUSD",
+    vToken: "0xBf762cd5991cA1DCdDaC9ae5C638F5B5Dc3Bee6E",
+    supplySpeed: "0",
+    borrowSpeed: "0",
+    oldSupplySpeed: emissionPerBlock("3.12").div(2).toString(),
+    oldBorrowSpeed: emissionPerBlock("3.12").div(2).toString(),
+  },
```
Comment (chechu) [inline vips/vip-245.ts:178]: Not needed. It was already set to zero in the [VIP-240](https://app.venus.io/#/governance/proposal/240?chainId=56)

File: simulations/vip-245/simulations.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  borrowSpeeds,
+  supplySpeeds,
+  vTokens,
+  vip245,
+} from "../../vips/vip-245";
+import COMPTROLLER_ABI from "./abi/Comptroller.json";
+
+const OLD_VAI_VAULT_RATE = parseUnits("125.00", 18).div(blocksPerDay);
+
+export const oldSupplySpeeds = Speeds.map(speed => speed.oldBorrowSpeed);
```
Comment (chechu) [inline simulations/vip-245/simulations.ts:21]:
```suggestion
export const oldSupplySpeeds = Speeds.map(speed => speed.oldSupplySpeed);
```

File: simulations/vip-245/simulations.ts [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+export const oldBorrowSpeeds = Speeds.map(speed => speed.oldBorrowSpeed);
+
+forking(35518397, () => {
```
Comment (chechu) [inline simulations/vip-245/simulations.ts:24]: I would update the block number to a value post VIP-240, when the XVS emission of TUSD were modified

File: vips/vip-245.ts [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+import { parseUnits } from "ethers/lib/utils";
+
+import { ProposalType } from "../src/types";
+import { makeProposal } from "../src/utils";
+
+export const blocksPerYear = 10512000;
+export const blocksPerDay = parseInt((blocksPerYear / 365).toString());
+
+export const UNITROLLER = "0xfD36E2c2a6789Db23113685031d7F16329158384";
+export const VAI_VAULT_RATE = parseUnits("62.50", 18).div(blocksPerDay);
```
Comment (chechu) [inline vips/vip-245.ts:11]: This is `2170138888888888`. If we multiply `2170138888888888 * 28800` we get `62499999999999974400`, that would be shown in the UI this way: `62.49`

It we round up, adding +1, we would have `2170138888888889`. If we multiply `2170138888888889 * 28800` we get `62500000000000003200`, that would be shown in the UI this way: `62.50`. So, I think I would round up this number, to improve their aspect in the UI and taking into account it's a minimal change in the value

File: simulations/vip-245/simulations.ts [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+        const supplyRate = await comptroller.venusSupplySpeeds(Speeds[i].vToken);
+        expect(supplyRate).to.be.closeTo(oldSupplySpeeds[i], parseUnits("0.0001", 18).toString());
+
+        const borrowRate = await comptroller.venusBorrowSpeeds(Speeds[i].vToken);
+        expect(borrowRate).to.be.closeTo(oldSupplySpeeds[i], parseUnits("0.0001", 18).toString());
+      });
+    }
+  });
+
+  testVip("VIP-244", vip245(), {
```
Comment (GitGuru7) [inline simulations/vip-245/simulations.ts:48]:
```suggestion
  testVip("VIP-245", vip245(), {
```

**Review by chechu — APPROVED** [review]

**Pull request closed without merging**